filters/resources.txt

```diff
@@ -2310,3 +2310,34 @@ damoh-defuser.js application/javascript
     var observer = new MutationObserver(cleanVideoAsync);
     observer.observe(document.documentElement, { childList: true, subtree: true });
 })();
+
+
+twitch-videoad.js application/javascript
+(function() {
+	var ourMediaPlayer = undefined;
+	Object.defineProperty(window, 'MediaPlayer', {
+		set: function(newMediaPlayer) {
+			if (typeof(ourMediaPlayer) !== 'undefined')
+				return;
```

Comment:
Why? I see no reason to return here, just patch new object and assign.

Reply:
I want to prevent patching the same object multiple times. I suppose `ourMediaPlayer === newMediaPlayer` is better. But I'll wait with further commits until I know whether the maintainers are in general fine with merging this PR.

Reply:
This is a style suggestion, it's ultimately up to you and gorhill: 

`typeof` is not a function, it is a keyword, to stay consistent with the style of other scriptlets, it should be: 
```
if (typeof ourMediaPlayer !== 'undefined') { return; }
```